sdk/python/kfp/components/pipeline_channel_test.py

```diff
@@ -34,15 +34,15 @@ def test_invalid_name(self):
                 ValueError,
                 'Only letters, numbers, spaces, "_", and "-" are allowed in the '
                 'name. Must begin with a letter. Got name: 123_abc'):
-            p = pipeline_channel.PipelineParameterChannel(
+            p = pipeline_channel.create_pipeline_channel(
```

Comment:
Using the same constructor function in test cases that is used in the library code, instead of constructing the `PipelineChannel`s directly